supplemental-ui/.htaccess

```diff
@@ -1,25 +1,10 @@
 DirectoryIndex index.html index.asciidoc
 
-# former path to antora path
-Redirect 301 /about/ /front/main/about/
-Redirect 301 /blogs/ /front/main/blogs/
-Redirect 301 /jackpot/ /front/main/jackpot/
-Redirect 301 /community/ /front/main/community/
-Redirect 301 /help/ /front/main/help/
-Redirect 301 /projects/ /front/main/projects/
-Redirect 301 /participate/ /front/main/participate/
-Redirect 301 /kb/ /tutorial/main/kb/
-Redirect 301 /tutorials/ /tutorial/main/tutorial/
-Redirect 301 /wiki/ /wiki/main/wiki/
```

Comment:
Same thing for all of these. If a user is sent to "/wiki" from a search engine, we want to redirect the users to "/wiki/main/wiki" instead. So we want to keep these redirections.